integration/spark/src/main/scala/org/apache/spark/sql/CarbonSession.scala

```diff
@@ -293,5 +293,10 @@ object CarbonSession {
     def merge(srcDS: Dataset[Row], expr: Column): MergeDataSetBuilder = {
       new MergeDataSetBuilder(ds, srcDS, expr, ds.sparkSession)
     }
+
+    def merge(srcDS: Dataset[Row], keyColumn: String, operationType: String): UpsertBuilder = {
```

Comment:
No need use merge method name and pass the operationType, you can directly name the method as 'upsert'

Reply:
ok, for the user exposed APIs, I will change to update, delete, upsert and insert. Internally we can call this merge method and make this private and keep other code same for better code handling. This is because internally we need to call the merge command class itself, so I thought we can do it this way.